src/merkle/store/tests.rs

```diff
@@ -36,13 +36,13 @@ fn test_root_not_in_store() -> Result<(), MerkleError> {
     let mtree = MerkleTree::new(VALUES4.to_vec())?;
     let store = MerkleStore::from(&mtree);
     assert_eq!(
-        store.get_node(VALUES4[0], NodeIndex::make(mtree.depth(), 0)),
-        Err(MerkleError::RootNotInStore(VALUES4[0])),
+        store.get_node(VALUES4[0].into(), NodeIndex::make(mtree.depth(), 0)),
+        Err(MerkleError::RootNotInStore(VALUES4[0].into())),
         "Leaf 0 is not a root"
```

Comment:
I wonder if we'd end up with fewer changes if `VALUES8` was `[RpoDigest; 8]` and then we'd have a helper function to do something like:
```Rust
let mtree = MerkleTree::new(words_to_digests(&VALUES4))?;
```

Reply:
just to clarify, you mean `digests_to_words` right?